carbon-design-system/carbon-addons-iot-react
# fix(value-card): fix unit position in certain situations

Closes #2903 

### This is opened against master to make it into 8.6

**Summary**

- Fixes the unit positioning in certain situations

**Change List (commits, features, bugs, etc)**

- add css var to adjust the wrapper width based on the presence of a unit

**Acceptance Test (how to verify the PR)**

- go to the [valueCard with data filters story](https://deploy-preview-2904--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-valuecard--data-filters) and change the size to small
- or follow the instructions in the original issue and confirm the units are correctly placed beside the value.

**Regression Test (how to make sure this PR doesn't break old functionality)**

- check other ValueCard stories for issues.



**Things to look for during review**
